pkg/network/port_test.go

```diff
@@ -120,11 +121,11 @@ func TestReadInitialUDPState(t *testing.T) {
 	err := exec.Command("testdata/setup_netns.sh", nsName).Run()
 	require.NoError(t, err, "setup_netns.sh failed")
 
-	l, err := net.ListenUDP("udp", &net.UDPAddr{})
+	l, err := nettestutil.StartServerUDP(t, net.ParseIP("fd00.1"), 0)
 	require.NoError(t, err)
 	defer func() { _ = l.Close() }()
 
-	l6, err := net.ListenUDP("udp6", &net.UDPAddr{})
+	l6, err := nettestutil.StartServerUDP(t, net.ParseIP("fd00::1"), 0)
```

Comment:
Ditto

Reply:
Equivalent ipv6 would be `::`

Reply:
does this work because the method parsing this is looking for colons to indicate ipv6, but then it picks a random addr because it's "empty"?

Reply:
The previous code would actually listen on all interfaces (inaddr_any, i.e. "0.0.0.0"). `::` would be the equivalent ipv6.